src/catalog/src/builtin.rs

```diff
@@ -6198,17 +6198,18 @@ pub static MZ_SHOW_CLUSTER_REPLICAS: Lazy<BuiltinView> = Lazy::new(|| BuiltinVie
     mz_catalog.mz_clusters.name AS cluster,
     mz_catalog.mz_cluster_replicas.name AS replica,
     mz_catalog.mz_cluster_replicas.size AS size,
-    statuses.ready AS ready
+    COALESCE(statuses.ready, false) AS ready 
 FROM
     mz_catalog.mz_cluster_replicas
         JOIN mz_catalog.mz_clusters
             ON mz_catalog.mz_cluster_replicas.cluster_id = mz_catalog.mz_clusters.id
-        JOIN
+        LEFT JOIN
```

Comment:
`mz_hydration_statuses` stores hydration statuses for installed dataflows, in the replica is not yet online it won't have installed dataflows so we need to left join here and do the coalesce above to coarse into a bool. "ready"=true now means online + hydrated, and "ready"=false means offline or not hydrated.